renderlib/graphics/gl/Font.cpp

```diff
@@ -0,0 +1,146 @@
+#include "Font.h"
+
+#include "Util.h"
+
+#define STB_TRUETYPE_IMPLEMENTATION
+#include "stb/stb_truetype.h"
+
+#include <stdio.h>
+
+static constexpr int s_textureSize = 512;
+
+GLuint
+generateFontTexture(unsigned char* temp_bitmap)
+{
+  // expand temp_bitmap to RGBA
+  unsigned char* expanded_bitmap = new unsigned char[s_textureSize * s_textureSize * 4];
+  for (int i = 0; i < s_textureSize * s_textureSize; ++i) {
+    expanded_bitmap[i * 4 + 0] = 255;
+    expanded_bitmap[i * 4 + 1] = 255;
+    expanded_bitmap[i * 4 + 2] = 255;
+    expanded_bitmap[i * 4 + 3] = temp_bitmap[i];
+  }
```

Comment:
Oh interesting, is the font texture only stored as an alpha/greyscale value?

Reply:
yep. Shader can apply color if it's ever needed.